### PR TITLE
[wave2-819] special bg video updates

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

**Comment by dpfitzsi** [comment]
Appears to be in the backlog and not connected to any current release.

File: sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_bg_video/templates/fieldable-panels-pane--spl_bg_video.tpl.php [hunk 1]
```diff
@@ -4,14 +4,22 @@
...
+        // loop video(s) if not disabled
+       $loop = 'loop';
+       if($field_disable_looping[0]['value'] == 1) {
+          $loop = '';
+       }
     }
 
+
+
     if (count($videos) > 0):
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_bg_video/templates/fieldable-panels-pane--spl_bg_video.tpl.php]: I would include a comment here that if there is more than one video we will now pass to the JavaScript below to create the loop.

Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_bg_video/templates/fieldable-panels-pane--spl_bg_video.tpl.php]: Hi @dpfitzsi, I added this comment as part of PR 440

**Comment by dpfitzsi** [comment]
This one has conflicts, can you get them resolved and then I can merge this one to develop.

**Comment by dpfitzsi** [comment]
PR has been merged in 440. Closing this PR.

**Pull request closed without merging**